CraftTweaker2-API/src/main/java/crafttweaker/CraftTweakerAPI.java

```diff
@@ -269,9 +269,10 @@ public static void registerClass(Class annotatedClass) {
             }
             if((annotation instanceof BracketHandler) && IBracketHandler.class.isAssignableFrom(annotatedClass)) {
                 try {
-                    IBracketHandler bracketHandler = (IBracketHandler) annotatedClass.newInstance();
+                    IBracketHandler bracketHandler = (IBracketHandler) annotatedClass.getDeclaredConstructor().newInstance();
```

Comment:
ohhhhhh it's because it's deprecated in Java 11, but I can't use Java 8 to run the linter because Red Hat only works with Java 11+. I'll undo it and put a SuppressWarnings there to satisfy the linter

Reply:
You can still use it, but then use `getConstructor()` instead of `getDeclaredConstructor()`

Basically `declared` includes private and protected.

Reply:
/shrug if it works fine in Java 8, might as well undo the change